cmd/completion.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const completionCmd = `Use "arkade completion SHELL" to generate SHELL completion for:
+    - bash
+    - zsh
+`
+
+func MakeShellCompletion() *cobra.Command {
+
+	completion := &cobra.Command{
+		Use:   "completion SHELL",
+		Short: "Output shell completion for the given shell (bash or zsh)",
+		Long: `
+Outputs shell completion for the given shell (bash or zsh)
+This depends on the bash-completion binary.  Example installation instructions:
```

Comment:
Can we download the bash-completion binary with arkade get?

Reply:
@alexellis I suppose we can. Do you prefer it that way?